src/components/modal/modal.component.js

```diff
@@ -63,7 +63,7 @@ class Modal extends React.Component {
   handleOpen() {
     this.listening = true;
     this.updateDataState();
-    this.onOpening; // eslint-disable-line no-unused-expressions
+    this.onOpening(); // eslint-disable-line no-unused-expressions
```

Comment:
Instead of defining an empty `onOpening` method we should override `handleOpen` in the subclass and call the `super()`.